Makefile

```diff
@@ -41,3 +41,11 @@ clean:
 
 generate:
 	go generate ./...
+
+docker_build:
+	docker build -t mistryd:latest .
+
+docker_run:
+	docker run --rm -p 8462:8462 -v /tmp/mistry:/var/lib/mistry -ti mistryd:latest
+
+run_mistryd: mistryd docker_build docker_run
```

Comment:
I chose this approach (makefile) instead of a `bin/mistryd` shell script (which was my initial intuition) because I think that having a single entry point for building and running all components of the project is proper.